go/runtime/host/sgx/common/common.go

```diff
@@ -181,9 +192,24 @@ func AttestationWorker(
 			logger.Error("failed to regenerate CapabilityTEE",
 				"err", err,
 			)
+
+			// Setup a retry ticker so we retry attestation faster than the configured interval.
+			if retryTicker == nil {
+				expBackoff := backoff.NewExponentialBackOff(
+					backoff.WithMaxElapsedTime(0), // Never stop.
+				)
+				retryTicker = backoff.NewTicker(expBackoff)
+			}
+
 			continue
 		}
 
+		// Clear retry ticker after successful attestation.
+		if retryTicker != nil {
+			retryTicker.Stop()
```

Comment:
We should probably also stop the ticker if we return and is not nil.

Reply:
Fixed.